test/util/core/export-async-behaviors-test.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+var referee = require("@sinonjs/referee");
+var assert = referee.assert;
+
+var exportAsyncBehaviors = require("../../../lib/sinon/util/core/export-async-behaviors");
+
+var methods = {
+    callsFake: function callsFake() {
+        return "1";
+    },
+    yieldsOn: function yieldsOn() {
+        return "2";
+    },
+    callsArg: function callsArg() {
+        return "3";
+    },
+    resolvesThisAsync: function resolvesThisAsync() {
+        return "4";
+    }
+};
+
+describe("util/core/exportAsyncBehaviors", function() {
+    it("should rename functions to Async names and should avoid creating another async versions of the newly added async methods", function() {
```

Comment:
I think it would be good to show both the positive and negative with tests.

```js
context("for methods with names starting with 'callsArg' or 'yields'", function(){
    it("should create an async version", function(){
    });
});

context("for methods with names not starting with 'callsArg' or 'yields'", function(){
    it("should not add any new methods", function(){
    });
});
```

Reply:
👍 Thanks for the idea showing both give clearer intention